src/Logging.XUnit/XUnitLogger.cs

```diff
@@ -264,28 +267,53 @@ private static string GetLogLevelString(LogLevel logLevel)
         private static void GetScopeInformation(StringBuilder builder)
         {
             var current = XUnitLogScope.Current;
-            string scopeLog;
-            int length = builder.Length;
 
+            var stack = new Stack<XUnitLogScope>();
             while (current != null)
             {
-                if (length == builder.Length)
-                {
-                    scopeLog = $"=> {current}";
-                }
-                else
+                stack.Push(current);
+                current = current.Parent;
+            }
+
+            var depth = 0;
+            string DepthPadding(int depth) => new string(' ', depth * 2);
```

Comment:
This could be `static`?

```suggestion
            static string DepthPadding(int depth) => new string(' ', depth * 2);
```